graph.go

```diff
@@ -160,8 +160,8 @@ type MutableGraph interface {
 
 	// Like EdgeBetween in Graph, AddUndirectedEdge adds an edge between two nodes.
 	// If one or both nodes do not exist, the graph is expected to add them. However,
-	// if the nodes already exist it should NOT replace existing nodes with e.Head() or
-	// e.Tail(). Overwriting nodes should explicitly be done with another call to AddNode()
+	// if the nodes already exist it should NOT replace existing nodes with e.From() or
+	// e.To(). Overwriting nodes should explicitly be done with another call to AddNode()
```

Comment:
Dot at the end?

BTW, has there been a conclusion on if AddNode() should overwrite existing nodes or not?